src/test/java/core/basesyntax/service/impl/ReportGeneratorImplTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReportGenerator;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportGeneratorImplTest {
+    private static final String HEADER = "fruit,quantity";
+    private static ReportGenerator reportGenerator;
+
+    @BeforeClass
+    public static void beforeAll() {
+        reportGenerator = new ReportGeneratorImpl();
+    }
+
+    @Test
+    public void createReport_Ok() {
```

Comment:
maybe you should add null tests and test for empty string if you logic check these cases